site/content/contribute/why-contribute/_index.md

```diff
@@ -0,0 +1,103 @@
+3. Create a PR on the `master` branch of the applicable repository. If there is an associated [Jira](https://mattermost.atlassian.net/issues/?jql=) or GitHub issue, your PR should begin with the issue ID (e.g.  `[MM-394]` or  `[GH-394]`). Our GitHub PR template will walk you through the creation of your PR.
+4. If you’re a community contributor, the team at Mattermost will handle most of the review process for you. You can wait for a reviewer to be assigned and for them to review your work. Generally the process works like you’d expect: they’ll make suggestions, you’ll implement them and discuss it in the PR comments. If you’re having trouble getting a reviewer to be assigned to your PR, take a look at GitHub’s suggested reviewers and reach out to them in the [Community workspace](http://community.mattermost.com).
+5. Here is an overview of how our review process works. If you’re a core contributor, you can manage the process yourself.
+
+    1. Start by adding the  `1: UX Review`,  `2: Dev Review`, and  `3: QA Review`  labels, as well as whichever are applicable from [the list of labels]({{< ref "/contribute/more-info/getting-started/labels" >}}).
+    2. Add a milestone or a cherry-picking label if the PR needs it. Most don’t and will simply ship once merged, but if there are upcoming milestones, some reviewers are going to prioritize reviews attached to those milestones. Note that adding a milestone is mandatory for bug fixes that must be cherry-picked.
+    3. Likely you know whose code you’re changing, so assign the appropriate PM or Designer first for a UX review. When in doubt, somebody is better than anybody, as they can always reassign. Once they give the OK, remove the `1: UX Review` label.
+    4. Request two core committers to review — one with experience solving that particular or in that particular language, and another with expertise in the domain of your changes. Don’t request the same people over and over again unless they’re the only people with the necessary expertise. Once they give the OK, remove the `2: Dev Review` label.
+    5. Request a QA tester from the team that owns the code that you’re changing. They’ll decide the necessary scope of testing. Make sure you’ve defined what the expected results are so they know when their tests are successful. Once they give the OK, remove the `3: QA Review` label. The QA tester may add the `QA Review Done` label as well.
+        1. Generally avoid doing QA testing yourself, unless it’s obvious that it’s not necessary. In any case, it likely won’t be obvious to onlookers, so clearly explain the rationale in the comments on the PR.
+    6. Once all outstanding requests are completed and all involved have approved of the PR, add the `4: Reviews Complete` if it hasn’t been already, merge the PR into `master`, and delete the old branch. If your pull request depends on other pull requests, consider assigning the  `Do Not Merge/Awaiting PR`  label to avoid merging prematurely.
```

Comment:
```suggestion
    f. Once all outstanding requests are completed and all involved have approved of the PR, add the `4: Reviews Complete` if it hasn’t been already, merge the PR into `master`, and delete the old branch. If your pull request depends on other pull requests, consider assigning the  `Do Not Merge/Awaiting PR`  label to avoid merging prematurely.
```